File: examples/app-primes/calc/primes.py
#!/usr/bin/python

import sys

def isPrime(n):
    if n<2:
        return False
    for i in range(2,n-1):
        if n%i==0:
            return False
    return True

def getNumPrimes(start, length):
	ret = 0
	for k in range(0, length):
		if isPrime(start + k):
			ret = ret + 1
	return ret

start=int(sys.argv[1])
length=int(sys.argv[2])

print(getNumPrimes(start,length))

File: examples/app-primes/misc/lang/php/primes.php
#!/usr/bin/php 

<?php
function isPrime($num) {
    if($num < 2) return false;
    for ($i = 2; $i < $num; $i++) {
        if ($num % $i == 0) {
          return false;
        }
    }
    return true; 
}

function getNumPrimes($start,$length){
  $ret = 0;
  for ($k = 0; $k < $length; $k++) {
    $n = $start + $k;
    if (isPrime($n)) {
      $ret++;
    }
  }
  return $ret; 
}

$start = $argv[1];
$length = $argv[2];

print(getNumPrimes($start,$length));
?>
